prac_08/silver_service_taxi_test.py

```diff
@@ -0,0 +1,16 @@
+"""
+CP1404/CP5632 Practical
+Silver Service Taxi
+"""
+
+from silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    """Testing the SilverServiceTaxi Class"""
+    my_taxi = SilverServiceTaxi("Fancy Taxi", 100, 2)
+    my_taxi.drive(18)
+    print(my_taxi, f"for a total of ${my_taxi.get_fare():.2f}")
```

Comment:
Your tests should have something in them to show whether they pass or fail - like we did with Guitar expected/actual.